docs/apache-airflow/installation/index.rst

```diff
@@ -0,0 +1,316 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+Installation
+------------
+
+.. contents:: :local:
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Installation
+    :hidden:
+
+    Prerequisites <prerequisites>
+    Dependencies <dependencies>
+    Supported versions <supported-versions>
+    Installing from sources <installing-from-sources>
+    Installing from PyPI <installing-from-pypi>
+    Setting up the database <setting-up-the-database>
+    Upgrading to Airflow 2 <upgrading-to-2>
+    Upgrade Check <upgrade-check>
```

Comment:
This feels like a separation Upgrading section instead of installation, wdyt?

Reply:
Why not. Let me try

Reply:
Nicer indeed (I also shortened some menus and changed it explicitly to "Upgrading 1.10 to 2".

![image](https://user-images.githubusercontent.com/595491/133696629-0e88e02b-cc4f-4c5f-b54f-9ac4c254a2ed.png)

![image](https://user-images.githubusercontent.com/595491/133696689-3e195f5f-1b06-4a8d-a707-433e94d25ca5.png)